tests/__fixtures__/data/filtercontext.ts

```diff
@@ -26,22 +45,8 @@ export const filterGroupContextValue: FilterGroupContextType = {
   searchValue: '',
   fieldId: '123',
   setSearchValue: () => null,
-  getCollapseProps: () => ({
-    id: 'id',
-    onTransitionEnd: () => null,
-    style: {},
-    'aria-hidden': 'false'
-  }),
-  getToggleProps: () => ({
-    disabled: false,
-    type: 'button',
-    role: 'role',
-    id: 'id',
-    'aria-controls': 'element',
-    'aria-expanded': 'true',
-    tabIndex: 0,
-    onClick: () => null
-  }),
+  getCollapseProps: getDefaultCollapseProps(),
```

Comment:
I thought `getCollapseProps` was a function. why do we assign it to the returned object here? shouldn't it be `getCollapseProps: getDefaultCollapseProps`, and similarly for `getToggleProps`?

Reply:
we resulted to using FilterGroupProvider instead